IBPSA/ThermalZones/ISO13790/BaseClasses/GainMass.mo

```diff
@@ -0,0 +1,50 @@
+within IBPSA.ThermalZones.ISO13790.BaseClasses;
+model GainMass "Mass node heat flow"
+
+  Real Am;
+  parameter Real At;
+  parameter Real HwinG;
+  parameter Real f_ms;
+  parameter Real Af;
+
+
+  Modelica.Blocks.Interfaces.RealInput intG
+    annotation (Placement(transformation(extent={{-140,-60},{-100,-20}})));
+  Modelica.Blocks.Interfaces.RealInput solG
+    annotation (Placement(transformation(extent={{-140,-100},{-100,-60}})));
+  Modelica.Blocks.Interfaces.RealOutput y=((Am/At)*(0.5*intG + solG))
```

Comment:
At (which now has become ATot) cannot be zero, as it is not a direct user input, but it is calculated as the sum of all surfaces facing the thermal zone. This can only be zero if all surfaces are zero, which basically cannot happen.

Reply:
That's right. It does not make sense to define a thermal zone without any surface areas. Maybe it would be useful to add an assertion for this case. In that way the user gets a more informative error message, if he does that mistake. (e.g. "At least one surface needs an area greater than zero").

Reply:
The multiplex in models such as `IBPSA.ThermalZones.ISO13790.Examples.HeatingCooling` is not needed, but it does not hurt. Either way is fine.